HacktoberfestProject.Tests/Services/TableServiceTests.cs

```diff
@@ -35,14 +36,15 @@ public TableServiceTests()
             };
 
             _tableContext = new Mock<ITableContext>(MockBehavior.Strict);
-            _sut = new TableService(_tableContext.Object);
+            _githubService = new Mock<IGithubService>(MockBehavior.Strict);
+            _sut = new TableService(_tableContext.Object,_githubService.Object);
         }
 
 
         [Fact]
         public void GivenNullUserRepository_Should_ThrowException()

```

Comment:
The name is not correct anymore. The first parameter is `TableContext`. 

_(sorry, this might be my bad 🙈)_